api/v1alpha4/clusterclass_types.go

```diff
@@ -53,6 +53,20 @@ type ClusterClassSpec struct {
 	Workers WorkersClass `json:"workers,omitempty"`
 }
 
+type ControlPlaneClass struct {
```

Comment:
Should we update the proposal accordingly?

Reply:
Yes, I can do that in here